README.md

```diff
@@ -66,6 +66,17 @@ Getting Mosh
   sure that a running process's environment variables are not readable by other
   users.  We have confirmed that this is the case on GNU/Linux, OS X, and
   FreeBSD.
+  
+Building Mosh From Source
+------------
+  On most operating systems, building is simple. Simply run the following commands as root:
```

Comment:
Even when doing that it's a very bad idea to run `make install` as root. You're installing software system-wide by hand - only the package manager can do that, so you can actually manage (e.g. uninstall) said software.

It's usually much better to let the package manager install the compiled files - e.g. `checkinstall` on Debian, writing a small `PKGBUILD` on Archlinux, etc.